.travis.yml

```diff
@@ -1,11 +1,16 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1
+  - 2.6.6
+  - 2.7.2
+  - 3.0
```

Comment:
Ah..., Ruby 3.0 matrix seems to fail. 

```console
Failures:
  1) Seory::Helper.rails_helper_methods defines shortcuts to rails helper method can call helper method
     Failure/Error: delegate(*args, {to: :helper})
     ArgumentError:
       Delegation needs a target. Supply a keyword argument 'to' (e.g. delegate :hello, to: :greeter).
     # /home/travis/.rvm/gems/ruby-3.0.0/gems/activesupport-6.1.3.1/lib/active_support/core_ext/module/delegation.rb:173:in `delegate'
     # ./lib/seory/helper.rb:7:in `rails_helper_methods'
     # ./spec/seory/helper_spec.rb:27:in `block (4 levels) in <top (required)>'
     # ./lib/seory/helper.rb:27:in `module_eval'
     # ./lib/seory/helper.rb:27:in `block in modulize'
     # ./lib/seory/helper.rb:24:in `initialize'
     # ./lib/seory/helper.rb:24:in `new'
     # ./lib/seory/helper.rb:24:in `modulize'
     # ./lib/seory/helper.rb:16:in `apply!'
     # ./spec/seory/helper_spec.rb:34:in `block (3 levels) in <top (required)>'
```

https://travis-ci.org/github/esminc/seory/jobs/765918053

I think Ruby 3.0 support can be opened by another PR. Can you revert this Ruby 3.0 matrix?

Reply:
>I think Ruby 3.0 support can be opened by another PR.

I was just thinking the same thing. :slightly_smiling_face: I'll revert that changes.